### DISCUSSION TITLE
Inspire and High Value Datasets ?

### DISCUSSION THREAD

**Discussion by dhdeangelis** [INSPIRE-MIF/helpdesk#160] [General]
Dear colleagues,

There is an ongoing discussion about how to describe High Value Datasets in metadata using the DCAT-AP standard:
[DCAT-AP 3.0](https://semiceu.github.io/DCAT-AP/releases/3.0.0/)
[DCAT-AP High Value Datasets](https://semiceu.github.io/uri.semic.eu-generated/DCAT-AP/releases/2.2.0-hvd/)

These documents do make reference to Inspire (for example: [10.2.2 Special cases](https://semiceu.github.io/uri.semic.eu-generated/DCAT-AP/releases/2.2.0-hvd/#special-cases)) but these references are of course not specific as to which additional elements might be needed within Inspire metadata in order to describe HVD.

I have not been able to find detailed discussions about this within the framework of Inspire. I wonder if this matter is being discussed actively and will appreciate any pointers to publicly available material.

Best regards

Hernán

**Comment by fabiovinci** [comment]
Dear @dhdeangelis,

there are no discussions, at the moment, about the use of specific metadata elements within INSPIRE metadata in order to describe HVD.

As you can see in this [MIG-T presentation](https://wikis.ec.europa.eu/pages/viewpage.action?pageId=99418952&preview=%2F99418952%2F99418937%2FMIG-T72_HVD_Technical.pdf), the "mapping" between INSPIRE data and the HVD Thematic scopes was done considering the INSPIRE data themes.

The new INSPIRE Geoportal will filter HVD by means of the INSPIRE data theme declared in the metadata:

![image](https://github.com/INSPIRE-MIF/helpdesk/assets/22212406/525a5fb9-8ea2-4ca8-874c-3597c205e524)

**Reply by dhdeangelis** [reply]
Thank you @fabiovinci for your answer. Could we then sum up by stating that Inspire datasets that: 1) comply with implementing rules, 2) have 'open' licenses, and 3) fall within the given themes; can be considered to fit within the framework of the open data legislation without any additional use of metadata elements or other means of labeling?

**Reply by fabiovinci** [reply]
At the moment, for the INSPIRE data that are in scope with the HVD Regulation, there are no additional metadata requirements.
In any case, the situation may evolve in the future.

**Reply by dhdeangelis** [reply]
OK! Thanks!
